airbyte-integrations/connectors/destination-snowflake/src/test-integration/kotlin/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeInternalStagingLowercaseDatabaseTypingDedupingTest.kt

```diff
@@ -38,4 +38,9 @@ class SnowflakeInternalStagingLowercaseDatabaseTypingDedupingTest :
     override fun testV1V2Migration() {
         super.testV1V2Migration()
     }
+
+    @Test
+    override fun testRemovingPKNonNullIndexes() {
```

Comment:
your call: want to remove these? (I don't care either way, but if we keep them it's probably worth adding a comment like `overridden so that we can run directly from intellij`)